docs/en/drivers/driver-cli.md

```diff
@@ -0,0 +1,74 @@
+## The Appium Driver Command Line Interface (CLI)
+
+Appium calls the bits of software that enable automation of a certain platform using a certain automation technology a "driver". There are drivers for iOS, Android, and other platforms. As of Appium 2.0, Appium does not download and install any drivers by default when you install Appium. There are too many drivers for this to be practical. Instead, you use a special command line interface to direct Appium to install and register drivers for you, to make them available for your use. This doc highlights the various commands available within this interface.
```

Comment:
It makes sense to mention that in the title, that we are talking about Appium 2.0+

Reply:
this won't be published though until 2.0 is published, right, so at that point won't it be moot?

Reply:
publishing Appium2.0 does not mean that people would stop using 1.0 immediately

Reply:
ok, fair enough!